app-backend/batch/src/main/scala/com/azavea/rf/batch/airflow/export/CreateExportDef.scala

```diff
@@ -0,0 +1,165 @@
+package com.azavea.rf.batch.airflow.export
+
+import java.util.UUID
+
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, Future}
+import scala.concurrent.duration._
+import scala.util._
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import cats.data._
+import cats.implicits._
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
+import com.amazonaws.services.elasticmapreduce.{AmazonElasticMapReduceClient}
+import com.amazonaws.services.elasticmapreduce.model.{AddJobFlowStepsRequest, StepConfig, HadoopJarStepConfig}
+import com.amazonaws.services.route53._
+import com.amazonaws.services.route53.model._
+import io.circe.generic.JsonCodec
+import io.circe.syntax._
+
+import com.azavea.rf.batch.JobNoEC
+import com.azavea.rf.batch.util._
+import com.azavea.rf.database.{Database => DB}
+import com.azavea.rf.database.tables._
+import com.azavea.rf.datamodel._
+
+case class CreateExportDef(exportId: UUID)(implicit val database: DB) extends JobNoEC {
```

Comment:
`JobNoEc` looks a bit useless, if that execution context implicit should not be used in this job (though probably it makes sense to parallelize export definitions writes on S3) it is better to remove execution context just from the `Job` trait and to add it in the lc8 import job impl.